app/app.go

```diff
@@ -478,15 +559,15 @@ func New(
 		slashingtypes.ModuleName, evidencetypes.ModuleName, stakingtypes.ModuleName, ibchost.ModuleName,
 		ibctransfertypes.ModuleName, authtypes.ModuleName, banktypes.ModuleName, govtypes.ModuleName,
 		genutiltypes.ModuleName, authz.ModuleName, feegrant.ModuleName, paramstypes.ModuleName, vestingtypes.ModuleName,
-		icatypes.ModuleName, icaauthmoduletypes.ModuleName, ibcfeetypes.ModuleName, chainmaintypes.ModuleName,
-		nfttypes.ModuleName, supplytypes.ModuleName,
+		icatypes.ModuleName, icaauthmoduletypes.ModuleName, wasm.ModuleName, ibcfeetypes.ModuleName,
```

Comment:
I saw "wasm" was the last in wasmd's app.go, not sure whether required or ok

Reply:
They mention that it only needs to come after IBC transfer module. In our case, I've put it after all the cosmos and ibc modules.